CMakeLists.txt

```diff
@@ -142,6 +143,7 @@ set(LAMINARD_CORE_SOURCES
     src/run.cpp
     src/server.cpp
     src/version.cpp
+    src/json.cpp
```

Comment:
I don't think it is necessary to break this out into a separate file - see later comments for explanation